rfq/order.go

```diff
@@ -128,7 +125,12 @@ func (c *AssetSalePolicy) CheckHtlcCompliance(
 
 	// Check that the HTLC amount is not greater than the negotiated maximum
 	// amount.
-	maxOutboundAmount := lnwire.MilliSatoshi(c.MaxAssetAmount) * c.AskPrice
+	maxAssetAmount := rfqmsg.NewBigIntFixedPoint(c.MaxAssetAmount, 0)
```

Comment:
See previous comment, we need to fetch the decimal display unit here and use it.